.yarn/sdks/eslint/bin/eslint.js

```diff
@@ -1,13 +1,13 @@
 #!/usr/bin/env node
 
 const {existsSync} = require(`fs`);
-const {createRequire, createRequireFromPath} = require(`module`);
+const {createRequire} = require(`module`);
```

Comment:
These `.yarn/*` file changes are generated from running `yarn dlx @yarnpkg/sdks`. They update the yarn plugins for the versions of prettier and typescript that this repo uses.